projects/Vula/test.nix

```diff
@@ -1,21 +1,81 @@
 {
   sources,
-  pkgs,
+  lib,
   ...
-}: {
+}: let
+  inherit (lib) recursiveUpdate;
+in {
   name = "vula";
 
-  nodes = {
-    server = {config, ...}: {
+  nodes = let
+    common = {
       imports = [
         sources.modules.default
         sources.modules."services.vula"
       ];
+
       services.vula.enable = true;
+      services.vula.openFirewall = true;
+
+      services.vula.logLevel = "DEBUG";
     };
+  in {
+    a = recursiveUpdate common {
+      services.vula.userPrefix = "fula";
+      services.vula.systemGroup = "zula";
+      services.vula.adminGroup = "vula-managers";
```

Comment:
vula doesn't have management; it has organized workers that conspire to achieve goals using the principle of least authority.

Reply:
`services.vula.adminGroup` is an option in the vula NixOS module. It specifies the name of the group that will be privileged to do anything useful with the vula cli. Both read and write (full) permissions. The default value is currently `vula-admins`. Is that okay? 

This particular line intentionally sets the value to something other than the default, in order to later assert that it actually applies.

Reply:
Naming things is hard, let me think about `vula-admins`.

The vula-managers line now makes sense to me in this context as an error, and I understand now that it fits the theme. Thank you for clarifying.

Reply:
Agreed `vula` is a reasonable group name for replacing `sudo` in the dbus policy. We may also make this change upstream at a later date.

Reply:
During the mob cake session we have decided to rename the `adminGroup` option name to `operatorsGroup`. And we settled for the default to be `vula-ops`